
Header header

uint8 status

float64 Vx
float64 Vy
float64 Vz
float64 Vv
float64 Heading
float64 Pitch
float64 Roll